nncf/onnx/graph/metatypes/onnx_metatypes.py

```diff
@@ -34,14 +34,14 @@ def get_subtypes(cls) -> List[Type[OperatorMetatype]]:
         return cls.subtypes
 
     @classmethod
-    def matches(cls, model: onnx.ModelProto, node: onnx.NodeProto) -> Optional[bool]:
+    def matches(cls, onnx_graph: ONNXGraph, node: onnx.NodeProto) -> Optional[bool]:
```

Comment:
Could you explain the benefits of migration to ONNXGraph?

Reply:
To be consistent that we use ONNXGraph as an interface to interact with ONNX model

Reply:
Then the name of the `ONNXGraph` abstraction confuses me.

Reply:
We have a ticket on removing ONNXGraph - 96982